chapters/statements.tex

```diff
@@ -26,7 +26,7 @@ \subsection{Initial Algorithm Sections}\label{initial-algorithm-sections}
 See \cref{initialization-initial-equation-and-initial-algorithm} for a description of both initial algorithm sections and
 initial equation sections.
 
-\subsection{Execution of an algorithm in a model}\label{execution-of-an-algorithm-in-a-model}
+\subsection{An algorithm in a model}\label{execution-of-an-algorithm-in-a-model}\label{an-algorithm-in-a-model}
```

Comment:
Are algorithm and model considered to be keywords in this case? If they are not, they should be capitalized. In the subsection title "Initial Algorithm" are not treated as keywords but in Section 8.6 title initial equation and initial algorithm are treated as keywords. At least, I assume they are because they are not capitalized.

Reply:
I would say non-keyword "algorithm" here. Will change

Reply:
Will also change 'model' and 'function', even though keywords; as it is more about the concepts.